### PR TITLE
adding Tran Bao Nguyen in student

### PR DESCRIPTION
**Pull request by bnguyenltt**
https://bnguyenltt.github.io/gci16.fossasia.org/

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -272,3 +272,11 @@
...
   facebook: sanchit.technogeek
   bio: 17, amateur Nodejs dev, roboticist, linux guru.
+
+- name: Tran Bao Nguyen
+  github: bnguyenltt
+  image: bnguyenltt.jpg
+  twitter: bnguyenltt
+  bio: 16, student, competitive programmer, interested in games and manga
+  facebook: bnguyenltt
+
```
Comment (jigyasa-grover) [inline _data/students.yml]: Remove trailing whitespace to fix build failure.

**Comment by jigyasa-grover** [comment]
Also squash all your commits into one and rename it to something meaningful, such as "Adding <-your-name-> in students/mentors section"

**Comment by niccokunzmann** [comment]
Can you add your entries to a random position in the file? May be you want to update this pull-request or open a new one. Several other people want to do the same task and if all add it to the end, several-1 people fail and need to do again. You can also talk to them, and collaborate.

**Comment by m1guelpf** [comment]
@bnguyenltt @jig08 @niccokunzmann Fixed Travis (partly). Gonna squash and merge now.

**Pull request merged**